server/session.ts

```diff
@@ -8,6 +8,8 @@ import * as Config from "./config";
 const SESSION_MAX_AGE_SECONDS = 12 * 60 * 60;
 
 const getRedisStore = () => {
+  if (Config.isDev) return undefined;
```

Comment:
Dette er bare for å kunne kjøre i finnfastlege på utviklermaskin uten noen redis.